### PR TITLE
MOYO solution

### PR DESCRIPTION
**Pull request by KirillStetsenko**
- [DEMO LINK](https://KirillStetsenko.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://KirillStetsenko.github.io/layout_moyo-header/report/html_report/)

- [ ] Header height is set in 1 place (for the links)
- [x] Content is vertically centered (for any header height)
- [x] CSS is used to show all letters in Uppercase (don't type them in HTML)
- [x] Logo is an image wrapped with a link
- [x] **CSS Variable** is used for a blue color
- [x] Pseudo-element is used for a blue line below the active link
- [x] Code follows all the [Code Style Rules ❗️](./checklist.md)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,14 +1,35 @@
...
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" >
     <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./style.css" >
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.google.com/specimen/Roboto" rel="stylesheet">
   </head>
   <body>
     <h1>Moyo header</h1>
```
Comment (artemforsoff) [inline src/index.html:14]: this shouldn't have been left

File: src/index.html [hunk 2]
```diff
@@ -1,14 +1,35 @@
...
+    <header class="nav">
+      <div class="nav-wraper">
+        <div class="nav-logo">
+          <a href="#"><img src="images/logo.png" alt="logo" ></a>
```
Comment (artemforsoff) [inline src/index.html:18]: the same 
all styles for `nav-logo` you can write for link and remove the extra wrapper

File: src/index.html [hunk 3]
```diff
@@ -1,14 +1,35 @@
...
+        </div>
+        <nav class="nav-nav">
```
Comment (artemforsoff) [inline src/index.html:20]: at this moment something went wrong 😅

File: src/index.html [hunk 4]
```diff
@@ -1,14 +1,35 @@
...
+          <ul class="nav-list">
+            <li class="nav-item"><a href="#" class="nav-link is-active">Apple</a></li>
+            <li class="nav-item"><a href="#" class="nav-link">Samsung</a></li>
+            <li class="nav-item"><a href="#" class="nav-link">Smartphones</a></li>
+            <li data-qa="hover" class="nav-item"><a href="#" class="nav-link">Laptops & Computers</a></li>
```
Comment (artemforsoff) [inline src/index.html:25]: 1. Add empty lines between multiline sibling blocks of HTML
2. Keep your attributes correctly formatted

check examples [here](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules#html-formatting)

File: src/style.css [hunk 5]
```diff
@@ -1,3 +1,78 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: bold;
```
Comment (artemforsoff) [inline src/style.css:4]: if you specify `font-weight` for the body tag, then all child tags will inherit this rule

File: src/index.html [hunk 6]
```diff
@@ -1,14 +1,35 @@
...
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" >
     <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./style.css" >
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.google.com/specimen/Roboto" rel="stylesheet">
```
Comment (artemforsoff) [inline src/index.html:11]: connect font correctly
you should choose the needed weights

File: src/style.css [hunk 7]
```diff
@@ -1,3 +1,78 @@
...
+  justify-content: center;
+  height: 40px;
+}
+
+.nav-list {
+  display: flex;
+}
+
+.nav-item {
+  margin-left: 20px;
```
Comment (artemforsoff) [inline src/style.css:37]: the first element has `margin-left`

File: src/style.css [hunk 8]
```diff
@@ -1,3 +1,78 @@
...
+  content: "";
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  height: 3px;
+  width: 100%;
+  background-color: var(--blue-color);
+}
+
+.nav2 {
```
Comment (artemforsoff) [inline src/style.css:76]: I'm guessing this is a style for a non-existent element

File: src/style.css [hunk 9]
```diff
@@ -1,3 +1,78 @@
...
+  position: relative;
+  color: var(--blue-color);
+}
+
+.is-active::after {
+  content: "";
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  height: 3px;
```
Comment (artemforsoff) [inline src/style.css:71]: in the design the dimensions are different and there is a roundness
зay attention to the design carefully

**Review by YegorKorenkov — APPROVED** [review]
You probably forgot to re-deploy Your code looks good, but to make sure it's correct, please re-deploy and re-request review again
image

**Review by YegorKorenkov — CHANGES_REQUESTED** [review]
.

**Pull request closed without merging**